docs/tutorials/hobby_calibration.rst

```diff
@@ -69,65 +71,68 @@ song, then the calibration-in-progress song, and finally plays the arming song a
 
-The first parameter to check is the Communication parameter in the General tab. When you calibrate a module, it will set its communication type to the protocol that was used to calibrate. This is only
-noticeable if the motor was previously on Autodetect. Check that the Communication setting matches the protocol you calibrated with. The figure below shows the value of this parameter after
-calibrating a module with the PWM protocol. If you do wish to switch to using another protocol, you can simply change the selected Communication protocol.
+The first parameter to check is the *Communication* parameter in the General tab. When you calibrate a module, it will set its communication type to the protocol that was used to 
+calibrate if *Communication* was previously set to *Autodetect*. Check that the *Communication* setting matches the protocol you calibrated with. If *Communication* was explicitly
+set to a value other than *Autodetect* prior to calibration, then its value should not change. The figure below shows the an example of the *Communication* parameter after
+calibrating a module with the Standard PWM protocol. If you do wish to switch to using another protocol, you can simply change the selected Communication protocol.
 
 .. figure:: ../_static/tutorial_images/calibration_tutorial/general_communication_calibrated.JPG
     :align: center
 
-    Communication Parameter in General Tab After Calibrating With PWM
+    Communication Parameter in General Tab After Calibrating With Standard PWM
 
 The next parameters of interest are in the Advanced tab. Open the Advanced tab, Clicking "Yes" when prompted. There are 4 relevant parameters here:
 
-    * **Hobby Calibrated Communication Protocol**: This indicates what protocol the module has been calibrated with. If the module has not been calibrated, this will be set to "No Calibration".
+    * **Hobby Calibrated Communication Protocol**: This indicates what protocol the module has been calibrated with. If the module has not been calibrated, this will be set to *No Calibration*.
       A protocol does not need to be calibrated to use it on the module. If a protocol is not calibrated the module will use its default endpoints for that protocol. Only one protocol can be calibrated at a
       time. This parameter is **read-only**, you cannot change the calibrated protocol from Control Center, you need to go through the calibration process with the desired protocol to set this.
-      This is a useful way to check that calibration was succesful. If this is at "No Calibration" after a calibration attempt, that indicates that the calibration did not complete for some reason.
+      This is a useful way to check that calibration was succesful. If this is at *No Calibration* after a calibration attempt, that indicates that the calibration did not complete successfully.
```

Comment:
succesful typo

Reply:
Is it? I checked in the dictionary: https://www.merriam-webster.com/dictionary/successful

Unless I'm looking at the wrong one, is this in reference to the successfully